### PR TITLE
Added webtrees issue #635

### PR DESCRIPTION
**Pull request by n8225**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ See [Contributing](#contributing).
       - [Web based file managers](#web-based-file-managers)
   - [Games](#games)
   - [Gateways](#gateways)
+  - [Geneaology](#geneaology)
```
Comment (Chouchen) [inline README.md]: should be `genealogy` if we're creating a genealogy category

**Comment by n8225** [comment]
Thanks for catching that.

**Comment by Kickball** [comment]
@nodiscc @AndyR207 @n8225 

I don't feel that genealogy warrants its own category, as there are not many selfhosted genealogy solutions and I don't imagine that genealogy is very popular. However it doesn't fit into any of the existing categories.

We could look at creating a misc/other category for solutions that don't fit into any categories and are not popular enough to warrant their own categories. How do you all feel about that?

**Comment by n8225** [comment]
I'm fine with either route.  I have found another project to add: [https://github.com/geneweb/geneweb](https://github.com/geneweb/geneweb)

**Comment by n8225** [comment]
Now that I've thought about it a little more,  I'm leaning towards the misc/other category.   If we go that route we should also set a minimum number of entries for a new category.

**Comment by nodiscc** [comment]
+1 please move this to a `Misc/Other` category. 3 items seem like a decent number to warrant creating a new section. You can also add this rule to the Contributing section.

**Comment by n8225** [comment]
Tried rebasing and it looks like I may have made a mess.  If need be I'll just delete this branch and copy paste the changes to a new branch.

**Comment by nodiscc** [comment]
Yes it  would be better to start a new PR. You can close this one when done.

**Pull request closed without merging**